### PR TITLE
OraDownloadData handle submissions with no file names

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** OraDownloadData blows up if it tries to handle an old file with no file name data

Testing:

Make an ORA response, then in your mysql shell, modify the submission by removing file sizes and file names. Don't just make an empty list, completely remove they keys. Then attempt to generate the Submission Files Summary Zip from the instructor dashboard 

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [ ] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [EDUCATOR-5365](https://openedx.atlassian.net/browse/EDUCATOR-5365)

FIY: @edx/masters-devs-gta

### CODE REVIEW

**Review by nsprenkle — COMMENTED** [review]
Code looks good and will circle back when I review functionality locally.

**Review by nsprenkle — APPROVED** [review]
Manual testing looks good.
